org.flatpak.popout3d.yml

```diff
@@ -0,0 +1,167 @@
+app-id: com.github.PopoutApps.popout3d
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+
+command: popout3d
+rename-desktop-file: popout3d.desktop
+rename-icon: popout3d_logo
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host:rw
+modules:
+
+  - name: python3-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} Pillow Pillow-PIL
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/3c/7e/443be24431324bd34d22dd9d11cc845d995bcd3b500676bcf23142756975/Pillow-5.4.1.tar.gz
+        sha256: 5233664eadfa342c639b9b9977190d64ad7aca4edc51a966394d7e08e7f38a9f
+      - type: file
+        url: https://files.pythonhosted.org/packages/5d/e1/d298934cd78984a5e8d616945349ed66a74b9df0337fb1941ce76b8a2b7b/Pillow-PIL-0.1dev.tar.gz
+        sha256: d80ee30805f10bf898df9ce7495f46e88990e8f90b5c2b917722d87526252b2c
+
+  # Hugin deps  
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.0.0.json
```

Comment:
But you are missing the submodule.

Reply:
Manifest renamed.
Could you explain what and from where please? Do you mean glew.json and glu-9.0.0.json? They are in the PopoutApps/popout3d repository. Should they be in the PopoutApps/flathub repository? Should they be in directories?

Reply:
No.

See https://github.com/flathub/flathub/wiki/App-Requirements#shared-modules